src/slam_karto.cpp

```diff
@@ -717,6 +673,20 @@ SlamKarto::mapCallback(nav_msgs::GetMap::Request  &req,
     return false;
 }
 
+void
```

Comment:
What was the reason to use a separate thread for visualization ? Having more control over the publish rate ?